### ISSUE TITLE
Lazy-Language-Loader

### ISSUE THREAD

**Issue by Kichura** [Madis0/fabulously-optimized#124]
### Mod name

lazy-language-loader

### Curseforge link

Edit by Madis0: https://www.curseforge.com/minecraft/mc-mods/lazy-language-loader

### Other links

https://github.com/ChachyDev/lazy-language-loader
https://modrinth.com/mod/lazy-language-loader

### What it does

Allows languages to be switched much faster without having to wait for a longer time.

### Why should it be in the modpack

In vanilla, changing languages will cause ALL resource pack-related materials to be reloaded,
While the mod itself makes it so that only the language texts get reloaded instead.

### Why shouldn't it be in the modpack

Not available in curseforge at this time, meaning this mod cannot be installed straight from curse's program.

### Categories

Performance optimization

### Additional details

Basically a very small mod but helps out a ton if players switch their game's languages often, For 1.16.5 support; the github page must be visited instead as modrinth's page only has 1.17.1 variant instead.

Demo of this mod:

https://user-images.githubusercontent.com/68134602/132931574-04fbd100-bda7-4658-b622-e45823ee9cc1.mp4

**Comment by RozeFound** [comment]
As a Russian, can approve this is really very useful thing. Time to time I switch lang in game from English to Russian and vice versa, and it's too boring to wait (in vanilla). I'm playing with this mod decent amount of time and doesn't seen any disadvantages.

**Comment by Madis0** [comment]
> if players switch their game's languages often

I need to know more about that - what kind of user would switch their languages often? [According to this image](https://minecraft.fandom.com/wiki/Language?file=Language_stats.jpg), less half of the users switch their language at all.

**Comment by RozeFound** [comment]
> I need to know more about that 

Don't you have any statistics in your modpack? Will be very useful to check it out instead of asking for feedback.

> I'd guess those that do are mostly "set-and-forget".

Even in this case, faster language change will be a pleasure.

**Comment by Madis0** [comment]
> Don't you have any statistics in your modpack? Will be very useful to check it out instead of asking for feedback.

Unfortunately no, hence I try to get feedback directly from the community.
  
> Even in this case, faster language change would be a pleasure.

Yes and no. Making things faster is good for sure, and I personally could be the target audience for this mod, however if maybe 1% of users switch their language more than once, it makes me wonder if this is worth it.

Although, now that I think of it, several users use clean profiles across upgrades (e.g. MultiMC) so the primary use case could be the _initial setup_, not the switch. (Because if constantly setting the language is annoying, people may skip it altogether.)

**Comment by RozeFound** [comment]
> if constantly setting the language is annoying, people may skip it altogether.

If people don't know English in enough manner (or at all), they never skip lang setting (as I do few years ago).

> Unfortunately no, hence I try to get feedback directly from the community.

Bad idea, as of 95% community don't give any feedback in most cases.

**Comment by Kichura** [comment]
Closing this as it got superior'd by other issue reference,

In case anything else is important here; please re-open this.
